app/controllers/application_controller.rb

```diff
@@ -1,2 +1,24 @@
 class ApplicationController < ActionController::Base
+  I18n::Backend::Simple.include(I18n::Backend::Pluralization)
+  
+  around_action :switch_locale
+
+  def default_url_options
+    { locale: I18n.locale }
+  end
+
+  def switch_locale(&action)
+    locale = params[:locale] || I18n.default_locale
+    I18n.with_locale(locale, &action)
+  end
+
+  private
+
+  def current_user
+    @current_user ||= Comments.find_by(id: session[:user_id])
+  end
+
+  def extract_locale_from_accept_language_header
```

Comment:
Why do you need this method?

Reply:
same as previous, I don't need this method )